scripts/create-kind-cluster-with-SSL-passthrough.sh

```diff
@@ -65,6 +71,27 @@ while (( $# > 0 )); do
   shift
 done
 
+
+###############################################################################
+# Prefetch the images required by nginx ingress
+###############################################################################
+prefetch_images=()
+prefetch_pids=()
+if [[ "$prefetch" == "true" ]] ; then
+  curl -s "$NGINX_INGRESS_URL" | sed 's/@sha256.*$//' > "$NGINX_INGRESS_FILE"
+  IFS=' ' read -r -a prefetch_images <<< "$(cat $NGINX_INGRESS_FILE | grep 'image: ' | awk '{print $2}' | uniq | tr '\n' ' ')"
+  for image in "${prefetch_images[@]}" ; do
+    echo -n "Prefetching image: \"${image}\"... "
+    docker pull "${image}" > /dev/null &
```

Comment:
Is this really an effective prefetch? The image gets used inside the kubeflex hosting cluster, not on the host machine.